<a href="https://colab.research.google.com/github/Sreya18/ML_Training/blob/main/Python/1st_program.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import os
print(os.getcwd())


C:\Users\KIIT\Desktop\ML_Dec


In [ ]:
print("Good Morning")

Good Morning


# This is the 1st program

In [ ]:
1+1


2

In [ ]:
print("Hello, How are you?")

Hello, How are you?


In [ ]:
str("5") + " there"

'5 there'

In [ ]:
"5"+"there"

'5there'

In [ ]:
a = 5
print(a)

5


In [ ]:
a='hi'
print(a)

hi


In [ ]:
A='10'
A=A+'1'
print(A)

101


In [ ]:
A=10
A=A+1
print(A)

11


In [ ]:
A='10'
B=A+'1'
print(A)
print(B)
C= int(A)+int(B)
print(C)

10
101
111


In [ ]:
print("knight's")

knight's


In [ ]:
print('knight"s')

knight"s


In [ ]:
a="How are u?"
len(a)

10

In [ ]:
print(a+a)

How are u?How are u?


In [ ]:
print(a*4)

How are u?How are u?How are u?How are u?


# **List**

In [ ]:
List=[1,'a',2,4]
print(List)
print(List[1])
List[2]=7
print(List)

[1, 'a', 2, 4]
a
[1, 'a', 7, 4]


## List is Mutable

In [1]:
nest=[1,2,3,[4,5,['target']]]

In [2]:
len(nest)

4

In [3]:
nest[3]

[4, 5, ['target']]

In [4]:
nest[3][2]

['target']

In [5]:
nest[3][2][0]

'target'

## **Tuple**

In [ ]:
T=('a',3,"How r u?")
T[1]=4
print(T)

TypeError: ignored

## Tuple is Immutable

In [ ]:
T=('a',3,"How r u?")
print(T)

('a', 3, 'How r u?')


In [ ]:
A={'a':1,'b':4,'c':6,'d':3}
A['b']
A['d']

3

In [ ]:
A={'a':1,'b':4,'c':6,'d':3}
print(A['b'])
print(A['d'])

4
3


In [ ]:
A='10'
type(A)

str

In [ ]:
c=int(A)
type(c)

int

In [ ]:
a=33
b=200

In [ ]:
if b>a:
    print("B is greater than a")
    a+=200
print(a)

B is greater than a
233


In [ ]:
if b>a:
    print("B is greater than a")
    a+=200
print(a)

233


### at 1st print will be printed since b>a, but if we run it again print will not be printed bcoz a is now incremented to 233

## **Sets**

In [6]:
{1,2,3}

{1, 2, 3}

In [7]:
s={1,2,3,1,2,3,1,1,1,3,3,2,2,2,1}

In [8]:
len(s)

3

In [9]:
s

{1, 2, 3}